app/uk/gov/hmrc/customs/emailfrontend/controllers/WhatIsYourEmailController.scala

```diff
@@ -70,7 +70,7 @@ class WhatIsYourEmailController @Inject()(actions: Actions, view: change_your_em
           case None => ??? //ToDo redirect to retry page
         }
       case SubscriptionDisplayResponse(None, Some(_)) => Future.successful(Redirect(routes.WhatIsYourEmailController.problemWithService()))
-      //case SubscriptionDisplayResponse(_, _) => ??? //ToDo
+      case SubscriptionDisplayResponse(None, None) => Future.successful(Redirect(WhatIsYourEmailController.verify()))
```

Comment:
can we use `routes.WhatIsYourEmailController` to avoid direct call to the function?